File: javascript/readStdin.js
#!/usr/bin/env node

/**
 * Usage: ./readStdin.js
 */

const readline = require("readline");

const input = readline.createInterface({
    input: process.stdin,
    output: process.stdout,
    terminal: false,
});

console.log("Press enter to send input...");
input.on('line', (line) => {
    // TODO: implementation
    console.log(`Input: ${line}`);
});

File: python/read_stdin.py
#!/usr/bin/env python

# Usage: ./read_stdin.py

import sys

def read_stdin():
    for line in sys.stdin:
        # TODO: implementation
        print("Input: " + line, end="")

if __name__ == "__main__":
    print("Press enter to send input...")
    read_stdin()
